bin/s2nd.c

```diff
@@ -467,13 +468,14 @@ int main(int argc, char *const *argv)
         {"stk-file", required_argument, 0, 'a'},
         {"no-session-ticket", no_argument, 0, 'T'},
         {"corked-io", no_argument, 0, 'C'},
+        {"max-conns", optional_argument, 0, 'X'},
         {"tls13", no_argument, 0, '3'},
         /* Per getopt(3) the last element of the array has to be filled with all zeros */
         { 0 },
     };
     while (1) {
         int option_index = 0;
-        int c = getopt_long(argc, argv, "c:hmnst:d:iTCX", long_options, &option_index);
+        int c = getopt_long(argc, argv, "c:hmnst:d:iTCX::", long_options, &option_index);
```

Comment:
What purpose does the extra colon add?

Reply:
The extra colon is an optional argument.